docs/articles_en/documentation/openvino-ir-format/operation-sets/operation-specs/movement/scatter-nd-update-15.rst

```diff
@@ -41,8 +41,9 @@ Operator ScatterNDUpdate-15 is an equivalent to following NumPy snippet:
         elif reduction == "min":
             func = min
         out = np.copy(data)
+        # Order of loop iteration is undefined.
         for ndidx in np.ndindex(indices.shape[:-1]):
-            out[indices[ndidx]] = func(out[indices[ndidx]], updates[ndidx])
+            out[tuple(indices[ndidx])] = func(tuple(out[indices[ndidx])], updates[ndidx])
```

Comment:
Is the `tuple` really needed in this example code?
Even tho the closing brackets seem to be broken ;)
```suggestion
            out[tuple(indices[ndidx])] = func(tuple(out[indices[ndidx]]), updates[ndidx])
```

Reply:
Without tuples, when lets say `indices[ndidx] == np.array([0, 0, 0])`, instead gathering and filling `out[0, 0, 0]` with update, it would use `integer array indexing` and try to gather `out[[0, 0, 0]]` being `np.array([out[0], out[0], out[0]])`.